### DISCUSSION TITLE
エディタの検索機能でマッチが0件だった場合に視覚的に教えてほしい

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#395] [要望・アイデア・不具合]
エディタの検索機能、基本的にとても便利なのですが、マッチする文字列がないときに何らかの形で「見つかりませんでした」という情報を視覚化してもらえると、さらに使い勝手が良くなりそうだなと思いました。
ご検討よろしくお願いします。

<img width="1488" alt="Screen Shot 2022-12-26 at 9 30 00" src="https://user-images.githubusercontent.com/1148320/209486023-dfcb7e2a-6704-4c93-ba97-eabcbdf9da2a.png">

**Comment by tanaka-asami415** [comment]
@JunichiIto 
ご意見ありがとうございます。
確かにそうですね！検討いたします！🙇
